src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -334,6 +334,7 @@ function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourceLoade
       return BINDING.js_to_mono_obj(Promise.resolve(0));
     }
 
+    const lazyResources = {};
```

Comment:
You could do something like `{ assemblies: undefined as string[] | undefined, pdbs: undefined as string[] | undefined }` to get some typing on this.

Reply:
Added types and a null check.